dockerfiles/packaging/guice/cassandra/package/etc/james/templates/mailetcontainer.xml

```diff
@@ -85,7 +85,7 @@
         <processor state="relay" enableJmx="true">
             <mailet match="All" class="RemoteDelivery">
                 <outgoingQueue>outgoing</outgoingQueue>
-                <delayTime>5000, 100000, 500000</delayTime>
+                <delayTime>5000, 100000, 23*500000</delayTime>
```

Comment:
Why 23?

Reply:
Because 1 + 1 + 23 = 25 which matches next line declaration, and prevent James to adapt to it, issuing a warning in the process...